zebra/label_manager.c

```diff
@@ -86,7 +86,23 @@ static int relay_response_back(void)
 			 strerror(errno));
 		return -1;
 	}
-	zlog_debug("Label Manager response received, %d bytes", size);
+
+	/* do not relay a msg that has nothing to do with LM */
+	switch (resp_cmd)
+	{
```

Comment:
coding style: `{` on the same line as `switch`

Reply:
thank you, I missed to pass it through clang-format-5, please excuse the inconvenience :-)